base/shell/explorer/trayntfy.cpp

```diff
@@ -121,125 +127,242 @@ class CTrayNotifyWnd :
         if (FAILED_UNEXPECTEDLY(hr))
             return FALSE;
 
+        /* Create the 'Show Desktop' button */
+        m_ShowDesktopButton.DoCreate(m_hWnd);
+        m_hwndShowDesktop = m_ShowDesktopButton.m_hWnd;
+
         return TRUE;
     }
 
     BOOL GetMinimumSize(IN OUT PSIZE pSize)
     {
-        SIZE szClock = { 0, 0 };
-        SIZE szTray = { 0, 0 };
+        SIZE clockSize = { 0, 0 };
+        SIZE traySize = { 0, 0 };
+        SIZE showDesktopSize = { 0, 0 };
 
         if (!g_TaskbarSettings.sr.HideClock)
         {
             if (IsHorizontal)
             {
-                szClock.cy = pSize->cy - 2 * TRAY_NOTIFY_WND_SPACING_Y;
-                if (szClock.cy <= 0)
+                clockSize.cy = pSize->cy;
+                if (clockSize.cy <= 0)
                     goto NoClock;
             }
             else
             {
-                szClock.cx = pSize->cx - 2 * TRAY_NOTIFY_WND_SPACING_X;
-                if (szClock.cx <= 0)
+                clockSize.cx = pSize->cx;
+                if (clockSize.cx <= 0)
                     goto NoClock;
             }
 
-            ::SendMessage(m_hwndClock, TNWM_GETMINIMUMSIZE, (WPARAM) IsHorizontal, (LPARAM) &szClock);
+            ::SendMessage(m_hwndClock, TNWM_GETMINIMUMSIZE, (WPARAM) IsHorizontal, (LPARAM) &clockSize);
```

Comment:
```suggestion
            ::SendMessage(m_hwndClock, TNWM_GETMINIMUMSIZE, (WPARAM)IsHorizontal, (LPARAM)&clockSize);
```

Reply:
Let's keep the style here, for the sake of smaller diff. (this PR already became overreviewed, style fixes can be done later)